mods/ra/weapons/missiles.yaml

```diff
@@ -90,16 +89,15 @@ HellfireAG:
 		Inaccuracy: 128
 		Image: DRAGON
 		HorizontalRateOfTurn: 10
-		RangeLimit: 4c819
+		RangeLimit: 8c819
```

Comment:
6c0.

Right now, all RangeLimits are 1.2*Range and not arbitrary numbers even if they seem as such at first sight.